docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:9
+
+RUN apt-get update -q && apt-get install -qy \
+  libssl-dev \
+  autoconf \
+  automake \
+  libtool \
+  python-setuptools python-dev \
+  curl \
+  build-essential \
+  pkg-config \
+  openjdk-8-jdk-headless \
+  git
+
+RUN cd $(mktemp -dt) && \
+  curl -sL 'https://github.com/facebook/watchman/archive/v4.9.0.tar.gz' | tar xzf - && \
+  cd watchman-* && \
+  ./autogen.sh && \
+  ./configure && \
+  make && \
+  make install
+
+RUN cd $(mktemp -dt) && \
+  curl -sL 'https://github.com/stephenh/mirror/archive/1.1.15.tar.gz' | tar xzf - && \
```

Comment:
Since we're using Quay now, can this `1.1.15` be a variable for the current branch/tag? Or will we even need this, e.g. Quay has repo access now so probably pulls down the source for the given branch/tag anyway.

Granted, makes sense why you needed this pre-Quay.

Reply:
Yes I agree, it should be using the local code now that it's in the repo!